strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/test/resources/org/carlspring/strongbox/authentication/api/impl/ldap/labet.properties

```diff
@@ -0,0 +1,19 @@
+strongbox.authentication.ldap.url=ldap://127.0.0.1:53389/dc=carlspring,dc=com
+!strongbox.authentication.ldap.managerDn=cn=manager,ou=users,dc=carlspring,dc=com
```

Comment:
I assume this is an invalid property...?

Reply:
Yes, I've removed it! Thanks! :)